content/content-strategy/identify-business-needs/who-to-talk/business-alignment-workshop/method.md

```diff
@@ -1,9 +1,13 @@
 ---
-layout: text/textblock
+layout: method/methodsummary
+preptime: 3 hrs
+people: 10+
+runtime: 2 hrs
+materials: materials.md
 ---
-
-Before starting the workshop, research the business areas. Have some idea of what you think the vision and mission of the agency could be. 
+<!-- Preparation --> 
```

Comment:
@sirleech these look like comments to me, in most of these header instances. Are they likely to be read out by screen readers?